dist/obs-server.spec

```diff
@@ -467,16 +492,38 @@ touch /srv/www/obs/api/log/production.log
 chown %{apache_user}:%{apache_group} /srv/www/obs/api/log/production.log
 
 %restart_on_update memcached
+%service_add_post obs-api-support.target
+%service_add_post obs-clockworkd.service
+%service_add_post obs-delayedjob-queue-consistency_check.service
+%service_add_post obs-delayedjob-queue-default.service
+%service_add_post obs-delayedjob-queue-issuetracking.service
+%service_add_post obs-delayedjob-queue-mailers.service
+%service_add_post obs-delayedjob-queue-project_log_rotate.service
+%service_add_post obs-delayedjob-queue-quick.service
+%service_add_post obs-delayedjob-queue-quick@.service
+%service_add_post obs-delayedjob-queue-releasetracking.service
+%service_add_post obs-sphinx.service
 # We need to touch the last_deploy file in the post hook
 # to update the timestamp which we use to display the
 # last deployment time in the API
 touch /srv/www/obs/api/last_deploy || true
 
 %postun -n obs-api
 %insserv_cleanup
+%service_del_postun obs-api-support.target
+%service_del_postun obs-clockworkd.service
+%service_del_postun obs-delayedjob-queue-consistency_check.service
+%service_del_postun obs-delayedjob-queue-default.service
+%service_del_postun obs-delayedjob-queue-issuetracking.service
+%service_del_postun obs-delayedjob-queue-mailers.service
+%service_del_postun obs-delayedjob-queue-project_log_rotate.service
+%service_del_postun obs-delayedjob-queue-quick.service
+%service_del_postun obs-delayedjob-queue-quick@.service
+%service_del_postun obs-delayedjob-queue-releasetracking.service
+%service_del_postun obs-sphinx.service
 %restart_on_update obsapisetup
 %restart_on_update apache2
-%restart_on_update obsapidelayed
```

Comment:
Does `%restart_on_update obs-api-support` stop everything that might be running from `obsapidelayed` from the previous version?

Reply:
@hennevogel:

> Does `%restart_on_update obs-api-support` stop everything that might be running from `obsapidelayed` from the previous version?

No, I was wrong. It was needed to check for upgrading in the `%post -n api` section. I added this check in a separated commit. Please have a look again.